hubble/src/indexer/mod.rs

```diff
@@ -29,9 +29,33 @@ pub struct Indexer<T: FetcherClient> {
     pub indexer_id: IndexerId,
     pub start_height: BlockHeight,
     pub chunk_size: usize,
+    pub finalizer_config: FinalizerConfig,
     pub context: T::Context,
 }
 
+#[derive(Clone, Debug, serde::Deserialize)]
+pub struct FinalizerConfig {
+    // how many blocks to wait until a block is considered finalized (ie. there should be no reorgs).
+    // compensates for height differences between rpcs
+    // default: 5
+    #[serde(default)]
+    pub delay_blocks: usize,
```

Comment:
this will default to 0, not 5 - you need to define a function and set `#[serde(default = "function_that_returns_default_value_you_want")]` and then use that same function in `FinalizerConfig::default` for consistency

Reply:
also good to use `NonZeroUsize` here probably?

Reply:
fixed the default. didn't do non-zero, as zero would be good value if we have an rpc that is consistent